Document detached workflows (#5222)
dapr/docs: daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-detached.md

---
type: docs
title: "Detached workflows"
linkTitle: "Detached workflows"
weight: 2600
description: "Schedule fire-and-forget workflows that run independently of their caller"
---

> Just like [child workflows]({{% ref "workflow-features-concepts.md#child-workflows" %}}), a workflow can schedule another workflow. With a _detached workflow_, however, the relationship is **fire-and-forget**: the calling workflow asks the runtime to start a new, fully independent workflow instance and then immediately continues with the rest of its own logic, without waiting for the detached workflow to complete.

Reviewer: ```suggestion
In addition to [child workflows]({{% ref "#child-workflows" %}}), a workflow can schedule another workflow as a _detached workflow_. With a _detached workflow_, however, the relationship is **fire-and-forget**: the calling workflow asks the runtime to start a new, fully independent workflow instance and then immediately continues with the rest of its own logic, without waiting for the detached workflow to complete.
```
